### PR TITLE
[14.0][ADD] hr_expense_cancel_confirm

### PR DESCRIPTION
**Pull request by Saran440**
This module add wizard when user click to cancel expense sheet, a confirmation wizard will be show, with reason as optional.

Depends:
- [ ] - https://github.com/OCA/hr-expense/pull/56

### CODE REVIEW

File: hr_expense_cancel_confirm/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{
+    "name": "HR Expense Cancel Confirm",
+    "version": "14.0.1.0.0",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "category": "Usability",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/hr-expense",
+    "depends": ["base_cancel_confirm", "hr_expense_cancel"],
+    "installable": True,
+    "maintainers": ["Saran Lim."],
```
Comment (kittiu) [inline hr_expense_cancel_confirm/__manifest__.py]: Not sure, should this be Saran440 ?

**Comment by kittiu** [comment]
@kittiu

File: hr_expense_cancel_confirm/__manifest__.py (hunk 1 above)
Comment (Saran440) [reply hr_expense_cancel_confirm/__manifest__.py]: Yes, you're right. I'm confused.

**Comment by kittiu** [comment]
@Saran440 please merge commit and rebase.

**Review by nimarosa — APPROVED** [review]
Looks good. I tested it and works like described. If someone wants to add comments please do, i can merge it in afternoon.

**Comment by Saran440** [comment]
@nimarosa please, wait because this module depend on https://github.com/OCA/hr-expense/pull/56

**Pull request closed without merging**